src/Components/3DV/SceneView.stories.tsx

```diff
@@ -16,6 +35,231 @@ export default {
+                    <FakePanel
+                        gizmoTransformItem={state.gizmoTransformItem}
+                        setGizmoTransformItem={setGizmoTransformItem}
+                    ></FakePanel>
+                )}
+            </div>
+        </div>
+    );
+};
+
+interface IFakePanelProps {
+    gizmoTransformItem: TransformInfo;
+    setGizmoTransformItem?: (gizmoTransformItem: TransformInfo) => void;
+}
+
+const FakePanel = (props: IFakePanelProps) => {
+    const { gizmoTransformItem, setGizmoTransformItem } = props;
+    const gizmoTransformItemDraftRef = useRef<TransformInfo>(null);
+
+    useEffect(() => {
+        if (gizmoTransformItem) {
+            gizmoTransformItemDraftRef.current = deepCopy(gizmoTransformItem);
+        }
+    }, [gizmoTransformItem]);
+
+    const onTransformChange = useCallback(
+        (event) => {
+            if (gizmoTransformItem) {
+                const { name, value } = event.target;
+                gizmoTransformItemDraftRef.current = deepCopy(
+                    gizmoTransformItem
+                );
+                const valueAsNumber = value;
+                switch (name) {
+                    case 'xPos':
+                        gizmoTransformItemDraftRef.current.position.x = valueAsNumber;
+                        break;
+                    case 'yPos':
+                        gizmoTransformItemDraftRef.current.position.y = valueAsNumber;
+                        break;
+                    case 'zPos':
+                        gizmoTransformItemDraftRef.current.position.z = valueAsNumber;
+                        break;
+                    case 'xRot':
+                        gizmoTransformItemDraftRef.current.rotation.x = valueAsNumber;
+                        break;
+                    case 'yRot':
+                        gizmoTransformItemDraftRef.current.rotation.y = valueAsNumber;
+                        break;
+                    case 'zRot':
+                        gizmoTransformItemDraftRef.current.rotation.z = valueAsNumber;
+                        break;
+                }
+                setGizmoTransformItem(gizmoTransformItemDraftRef.current);
+            }
+        },
+        [gizmoTransformItem]
+    );
+
+    const onTransformBlur = useCallback(() => {
+        setGizmoTransformItem({
+            position: {
+                x: Number(gizmoTransformItem.position.x),
+                y: Number(gizmoTransformItem.position.y),
+                z: Number(gizmoTransformItem.position.z)
+            },
+            rotation: {
+                x: Number(gizmoTransformItem.rotation.x),
+                y: Number(gizmoTransformItem.rotation.y),
+                z: Number(gizmoTransformItem.rotation.z)
+            }
+        });
+    }, [gizmoTransformItem]);
+
+    return (
+        <div style={{ padding: '20px' }}>
+            <Stack tokens={{ childrenGap: 12 }} horizontal>
+                <TextField
+                    label="X Position: "
```

Comment:
just to keep in mind, we might need to localize the labels when you consume this component (maybe you already did :))

Reply:
oh yeah I actually wasn't even thinking this component would be consumed; just used as a placeholder for now -- in the actual tab it will probably be more like X: Y: Z: with a "Position" label on top or something

Reply:
done